packages/core/src/common/message-translation.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2017 Ericson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+const Globalize = require("globalize");
+const localeRequire = require("../../i18n/locale.json");
+
+export class MessageTranslation {
+
+    static localize(selection: string, defaultSelection: string): string {
+        try {
+            return Globalize.formatMessage(selection);
+        } catch (Error) {
+            // If the language definition is not found, use the default selection string
+            console.log("----- using default value for: " + selection + " to be " + defaultSelection);
```

Comment:
We would need to use the logger for that log but it may be problematic since the class is static...
Not sure what the best option is here... 
We need to investigate if we can still use the logger in a static class